docs/manual-install.md

```diff
@@ -0,0 +1,149 @@
+# 🛠 Manual Configurations #
+
+This guide demonstrates steps required to enable the KMS Plugin for Key Vault in an existing cluster.
+
+1. Create a Keyvault
+
+    If you're bringing your own keys, skip this step.
+
+    ```bash
+    KEYVAULT_NAME=k8skv
+    RG=mykubernetesrg
+    LOC=eastus
+
+    # create resource group that'll contain the keyvault instance
+    az group create -n $RG -l $LOC
+    # create keyvault
+    az keyvault create -n $KV_NAME -g $RG
+    # create key that will be used for encryption
+    az keyvault key create -n k8s --vault-name $KV_NAME --kty RSA --size 2048
+    ```
+
+1. Give the cluster identity permissions to access the keys in keyvault
+
+    The KMS Plugin uses the cluster service principal or managed identity to access the keyvault instance.
+
+    With the cluster identity, assign the following permissions:
+
+    ```bash
+    az keyvault set-policy -n $KEYVAULT_NAME --key-permissions decrypt encrypt --spn <YOUR SPN CLIENT ID>
+    ```
+
+1. Deploy the KMS Plugin
+
+    For all Kubernetes master nodes, add the static pod manifest to `/etc/kubernetes/manifests`
+
+    ```yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: azure-kms-provider
+      namespace: kube-system
+      labels:
+        tier: control-plane
+        component: azure-kms-provider
+    spec:
+      priorityClassName: system-node-critical
+      hostNetwork: true
+      containers:
+        - name: azure-kms-provider
+          image: mcr.microsoft.com/oss/azure/kms/keyvault:v0.0.11
+          imagePullPolicy: IfNotPresent
+          args:
+            - --keyvault-name=${KV_NAME}                            # [REQUIRED] Name of the keyvault
+            - --key-name=${KEY_NAME}                                # [REQUIRED] Name of the keyvault key used for encrypt/decrypt
+            - --key-version=${KEY_VERSION}                          # [REQUIRED] Version of the key to use
+            - -v=1
+          ports:
+            - containerPort: 8787
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8787
+            failureThreshold: 2
+            periodSeconds: 10
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 4
```

Comment:
These are current defaults based on the manifests in aks-engine. We'll update this to a more concrete value after load testing: https://github.com/Azure/kubernetes-kms/issues/79